release.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+VERSION=$(git describe --abbrev=0 --tags)
+ARCH="${ARCH:-64}"
+LDC_FLAGS=()
+unameOut="$(uname -s)"
+case "$unameOut" in
+    Linux*) OS=linux; LDC_FLAGS=("-flto=full" "-linker=gold" "-static") ;;
+    Darwin*) OS=osx; LDC_FLAGS+=("-L-macosx_version_min" "-L10.7" "-L-lcrt1.o"); ;;
```

Comment:
Nitpick, I would prefer "macOS" over `osx`, since this is what the OS is called now.

Reply:
The naming is consistent with Dub's and LDC's releases:

https://github.com/dlang/dub/releases
https://github.com/ldc-developers/ldc/releases

Reply:
Yes, please @wilzbach. He right about the terminology. osx is a serie of  Mac OS versions.

Reply:
But that's true that even for pre-defined D version identifiers osx is used....

Reply:
Well, to be honest I don't feel strongly about it as long as we finally get the automated release builds working. We have been pushing this off for far too long ;-)

Reply:
> The naming is consistent with Dub's and LDC's releases:
But that's true that even for pre-defined D version identifiers osx is used....

That doesn't mean it's correct :smiley:. The predefined version identifier was arbitrary chosen by Walter even though `darwin` already existed in other compilers. Back then the official name was "Mac OS X", so naming it "OSX" was neither correct.